#!/usr/bin/env python
PACKAGE = "playground_builder"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("attentional_span",    int_t,    0, "Minimum duration (ms) of visual focus for an object to be considered attended to", 1000,  200, 5000)

exit(gen.generate(PACKAGE, "estimate_focus", "estimate_focus"))
